src/HRQLS/Controllers/Food/Sanitation.php

```diff
@@ -60,6 +60,6 @@ public function main(Request $req, Application $app)
             }
         }
 
-        return new Response(json_encode($sanitationdata), 201);
+        return new Response($_GET['callback'] . '('.json_encode($sanitationdata).')', 201);
```

Comment:
I'm going to let it slide this one time but we should have a comment explaining why we do these funkiness.